doc/rex-spec-v1.md

```diff
@@ -426,3 +427,22 @@ the scalar. The scale vector contains the scale values in each direction given i
 The `geometryId` can be zero which means that no geometry should be displayed. This indicates that
 the scenenode is an intermediate node in the scenegraph. All leafnodes in the scenegraph have to
 contain geometry information.
+
+#### Data Type Path (9)
+
+This data block can be used to describe a 3D path. A path is a container for 3D points, their
```

Comment:
Track ..... A track is a sequence of a 3D position and orientation of an AR device. The orientation is stored as a normalized normal vector of the device's LookAt vector.
The timestamp is using the UNIX time, the number of seconds elapsed since January 1, 1970 UTC.